app/public/js/location_search_map.js

```diff
@@ -78,11 +78,22 @@ function initMap() {
 	locations.forEach(function(location) {
 
 		var myLatLng = new google.maps.LatLng(location.latitude,location.longitude);
-
+		var markerIcon = '/images/maps/less_recommend.png'; // default for less than 40 rating
+		
+		if(location.rating>39){
```

Comment:
Pretty minor but can you put a space between "if" and the "("?  It would be nice to put spaces around the operators too like ">".